missing_locales.py

```diff
@@ -6,34 +6,55 @@
 """
 import argparse
 import json
+import sys
 from urllib.parse import quote as urlquote
 from urllib.request import urlopen
 
-IGNORE = ["templates", "configs"]
 
 def retrieve_pontoon_locales(project):
     query = f'{{project(slug:"{project}"){{name,localizations{{locale{{code}}}}}}}}'
     url = f"https://pontoon.mozilla.org/graphql?query={urlquote(query)}"
-    response = urlopen(url)
-    json_data = json.load(response)
 
-    locale_list = []
-    for locale in json_data['data']['project']['localizations']:
-        locale_list.append(locale['locale']['code'])
-    return locale_list
+    try:
+        response = urlopen(url)
+        json_data = json.load(response)
+        if "errors" in json_data:
+            sys.exit(f"Project {project} not found in Pontoon.")
+
+        locale_list = []
+        for locale in json_data["data"]["project"]["localizations"]:
+            locale_list.append(locale["locale"]["code"])
+        locale_list.sort()
```

Comment:
No issue, just easier to parse the output. In this specific case, you could even sort only the last pass (missing locales), since the rest is not exposed to "humans".